src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -66,18 +67,16 @@ function ProductGallery({ title }: Props) {
       {next !== false && (
         <>
           <GatsbySeo linkTags={[{ rel: 'next', href: next.link }]} />
-          <a
-            data-testid="show-more"
+          <ShowMoreLink
```

Comment:
@eduardoformiga actually, I don't think we need to create a new component just for that.
The only issue was because of the style. I talked with renatinha. Let's keep the default version (it doesn't need the extra padding on the left/right sides) 🤙 

 ``<LinkButton .... variant="secondary">
     Load more products
  </LinkButton>``

Reply:
Yeahh, Agree with you.
Let's change to keep it simple :)